frontend/src/v5/ui/components/shared/userMenu/editProfileModal/editProfilePersonalTab/editProfilePersonalTab.component.tsx

```diff
@@ -195,7 +212,7 @@ export const EditProfilePersonalTab = ({
 				</FormSelect>
 			</TabContent>
 			<FormModalActions>
-				<ModalCancelButton onClick={onClickClose} />
+				<ModalCancelButton disabled={!onClickClose} onClick={onClickClose} />
```

Comment:
This disable is kind of weird. There is some semantics lost here, I  know its because the user has missing data but is not what the code is describing here.